recipes/functools32/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: functools32
+  version: "3.2.3.2"
+
+source:
+  fn: functools32-3.2.3-2.tar.gz
+  url: https://pypi.python.org/packages/source/f/functools32/functools32-3.2.3-2.tar.gz
+  md5: 09f24ffd9af9f6cd0f63cb9f4e23d4b2
+
+requirements:
+  build:
+    - python 2.7.*
```

Comment:
Whilst this is the natural syntax for declaring this kind of thing, conda-build has made it quite hard to do this successfully in this form. In actual fact, this works find, but wouldn't work if we wanted to support two Python versions. In that case we would have to do:

```
build:  # [not py27]
   skip: True  # [not py27]
requirements:
   build:
      - python
   run:
      - python
```

As I said, this is academic for this particular recipe, but is worth stating none-the-less.